connector/src/main/java/org/geysermc/connector/entity/ItemEntity.java

```diff
@@ -55,4 +73,50 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
 
         super.updateBedrockMetadata(entityMetadata, session);
     }
+
+    @Override
+    public void moveRelative(GeyserSession session, double relX, double relY, double relZ, Vector3f rotation, boolean isOnGround) {
+        Vector3f movePosition = Vector3f.from(position.getX() + relX, position.getY() + relY, position.getZ() + relZ);
+        boolean OnGroundChange = onGround != isOnGround;
+
+        setRotation(rotation);
+        setOnGround(isOnGround);
+
+
+        if (position.distanceSquared(movePosition) > 0.25f || OnGroundChange)
```

Comment:
Could you follow the formatting that is presented throughout the rest of the project? For example, `if (this.position.distanceSquared(position) > 0.25f || OnGroundChange) {`